### DISCUSSION TITLE
Add an ebook reader or API that allows to type test an ebook

### DISCUSSION THREAD

**Discussion by schuster-rainer** [monkeytypegame/monkeytype#2518] [Ideas] [answered]
I would like to combine. Touch type testing and reading. So why not typetest a epub or PDF?

And example reader could be https://github.com/edrlab/thorium-reader

**Comment by Miodec** [comment, marked as answer]
You can simply copy the text and paste it into custom mode.

**Comment by schuster-rainer** [comment]
Yes, that is exactly what I am doing right now. I still think it would be nice to have the possibility to just use monkeytype as a reader. store progress and be able to continue where left off. Maybe that's sufficient. When i copy pastd a complete ebook into the custom text but then have some features where i pause and continue and have something like "pages" for better orientation. I just need to look at all the features first i guees. And have a look what's already there.
